CONTRIBUTING.md

```diff
@@ -44,6 +44,11 @@ if you fix an issue, please add 'Fixes #<issue_number> to the short description
 
 ### Push your changes
 
-Push changes to your repo and send a [pull request]
+Push changes to your repo 
+
+if you update Library, framework or third parties 
+Clone your repo in a different folder and compile+run your new code. This process will ensure that you not missing any file.
```

Comment:
I am not an native speaker but i would suggest to change the last sentence to:
"This process ensures that you do not forget any file."

Reply:
done